test/opengl/inline_reduction.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Halide.h"
+
+using namespace Halide;
+
+int main() {
+
+    // This test must be run with an OpenGL target
+    const Target &target = get_jit_target_from_environment();
+    if (!target.has_feature(Target::OpenGL)) {
+        fprintf(stderr, "ERROR: This test must be run with an OpenGL target, e.g. by setting HL_JIT_TARGET=host-opengl.\n");
+        return 1;
+    }
+
+    Func f;
+    Var x, y, c;
+    RDom r(0, 10);
+    f(x, y, c) = sum(r);
+    f.bound(c, 0, 3).glsl(x, y, c);
+
+    f.realize(100, 100, 3);
```

Comment:
IMHO WBN to have at least a comment indicating what the failure mode would be that this test is checking for.  WBN also to include at least a sanity check that it has generated the correct result, rather than simply not generated an internal error.